packages/salesforcedx-vscode-apex/README.md

```diff
@@ -43,10 +43,13 @@ To jump to the location of a definition, right-click the item and select **Go to
 ![Previewing, viewing, and jumping to a definition](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/images/apex_go_to_definition.gif)
 
 ## Find All References to Types
```

Comment:
We should change the title from "Find All References to Types" to "Find All References". The "type" seems unnecessary since we can actually find reference to more than just types now.